x-pack/docs/en/security/authentication/user-lookup.asciidoc

```diff
@@ -0,0 +1,57 @@
+[role="xpack"]
+[[user-lookup]]
+=== Looking up users without authentication
+
+{es} <<realms,realms>> exist primarily to support
+<<setting-up-authentication,user authentication>>. 
+Some realms authenticate users via a password (for example the
+<<native-realm,`native`>> and <<ldap-realm,`ldap`>> realms) and other realms use
+more complex authentication protocols (for example the <<saml-realm,`saml`>> and
+<<oidc-realm,`oidc`>> realms).
+In each case, the _primary_ purpose of the realm is to establish the identity of
+the user who has made a request to the {es} API.
+
+However, some {es} features need to _lookup_ a user without using their credentials.
+
+- The <<run-as-privilege,`run_as`>> feature executes requests on behalf of
+  another user. An authenticated user with `run_as` privileges can perform
+  requests on behalf of another unauthenticated user.
+
+- The <<authorization_realms,delegated authorization>> feature links two realms
+  together so that a user who authenticates against one realm can have the roles
+  and metadata associated with a user from a different realm.
+
+In each of the above cases the second user must be resolved without directly
+authenticating. The ability for a realm to lookup a user by username is an
+optional operation that is not supported by all realms.
+
+The reserved, <<native-realm,`native`>> and <<file-realm,`file`>> realms always
+support user lookup.
+
+The <<ldap-realm,`ldap`>> realm supports user lookup when the realm is configured
+in <<ldap-realm-configuration,_user search_ mode>>. User lookup is not support
+when the realm is configured with `user_dn_templates`.
+
+User lookup support in the <<active-directory-realm,`active_directory`>> realm
+requires that the realm be configured with a <<ref-ad-settings,`bind_dn`>> and a
+bind password.
```

Comment:
```suggestion
* The reserved, <<native-realm,`native`>> and <<file-realm,`file`>> realms always 
support user lookup.
* The <<ldap-realm,`ldap`>> realm supports user lookup when the realm is configured
in <<ldap-realm-configuration,_user search_ mode>>. User lookup is not support
when the realm is configured with `user_dn_templates`.
* User lookup support in the <<active-directory-realm,`active_directory`>> realm
requires that the realm be configured with a <<ref-ad-settings,`bind_dn`>> and a
bind password.
```

Reply:
I think that using a bulleted list to group these items would be good for readability.